src/bilder/images/concourse/deploy.py

```diff
@@ -168,7 +172,7 @@
 }
 
 # Install Concourse
-install_baseline_packages()
+install_baseline_packages(packages=["curl", "btrfs-progs"])
```

Comment:
Why btrfs?

Reply:
Because it's the recommended volume filesystem and because the fallback default of overlay has performance issues.